tests/bgp/test_bgp_stress_link_flap.py

```diff
@@ -23,6 +23,13 @@
 fanout_flap_count = 0
 neighbor_flap_count = 0
 
+LOOP_TIMES_LEVEL_MAP = {
+    'debug': 60,
+    'basic': 600,
```

Comment:
maybe we can reduce execution time for basic level

Reply:
Sure, will modify the basic level to 300s. then it will take 600s total in PR test, and 1200s in smoke test.

Reply:
I remember for PR test, we use the debug level, is it correct? if so, it would only take 120s for PR test. what about smoke test, do we have the default level? if no, it should use basic level, will take 1200s for smoke test.